data/js/server/teleporter/teleporter_solen_hive.js

```diff
@@ -0,0 +1,332 @@
+	"5711,1954": [5657, 1957],
+	"5712,1954": [5657, 1957],
+	"5713,1954": [5657, 1957],
+
+	//E leads to center area
+	"5700,1929": [5784, 1927],
+	"5701,1929": [5784, 1927],
+	"5702,1929": [5784, 1927],
+	"5700,1930": [5784, 1927],
+	"5701,1930": [5784, 1927],
+	"5702,1930": [5784, 1927],
+	"5700,1931": [5784, 1927],
+	"5701,1931": [5784, 1927],
+	"5702,1931": [5784, 1927],
+
+	//E leads back to area C
+	"5784,1929": [5657, 1957],
+	"5785,1929": [5657, 1957],
+	"5786,1929": [5657, 1957],
+	"5784,1930": [5657, 1957],
+	"5785,1930": [5657, 1957],
+	"5786,1930": [5657, 1957],
+	"5784,1931": [5657, 1957],
+	"5785,1931": [5657, 1957],
+	"5786,1931": [5657, 1957],
+
+	//Area D
+	//B leads to level 2
+	"5913,1893": [5815, 2013],
+	"5914,1893": [5815, 2013],
+	"5915,1893": [5815, 2013],
+	"5913,1894": [5815, 2013],
+	"5914,1894": [5815, 2013],
+	"5915,1894": [5815, 2013],
+	"5913,1895": [5815, 2013],
+	"5914,1895": [5815, 2013],
+	"5915,1895": [5815, 2013],
+
+	//B leads back to level 1
+	"5814,2015": [5914, 1891],
+	"5815,2015": [5914, 1891],
+	"5816,2015": [5914, 1891],
+	"5814,2016": [5914, 1891],
+	"5815,2016": [5914, 1891],
+	"5816,2016": [5914, 1891],
+	"5814,2017": [5914, 1891],
+	"5815,2017": [5914, 1891],
+	"5816,2017": [5914, 1891],
+
+	//F leads to center area
+	"5876,1891": [5808, 1908],
+	"5877,1891": [5808, 1908],
+	"5878,1891": [5808, 1908],
+	"5876,1892": [5808, 1908],
+	"5877,1892": [5808, 1908],
+	"5878,1892": [5808, 1908],
+	"5876,1893": [5808, 1908],
+	"5877,1893": [5808, 1908],
+	"5878,1893": [5808, 1908],
+
+	//F leads back to area D
+	"5809,1905": [5914, 1891],
+	"5810,1905": [5914, 1891],
+	"5811,1905": [5914, 1891],
+	"5809,1906": [5914, 1891],
+	"5810,1906": [5914, 1891],
+	"5811,1906": [5914, 1891],
+	"5809,1907": [5914, 1891],
+	"5810,1907": [5914, 1891],
+	"5811,1907": [5914, 1891],
+	"5809,1908": [5914, 1891],
+	"5810,1908": [5914, 1891],
+	"5811,1908": [5914, 1891],
+};
+
+// Exit coordinates
+var exitMap = {
+	// Minoc Exit
+	"5918,1794": [2607, 763],
+	"5919,1794": [2607, 763],
+	"5918,1795": [2607, 763],
+	"5919,1795": [2607, 763],
+	// Trinsic Exit
+	"5655,2018": [1689, 2789],
+	"5656,2018": [1689, 2789],
+	"5655,2019": [1689, 2789],
+	"5656,2019": [1689, 2789],
+	// Compassion Exit
+	"5919,2021": [1723, 814],
+	"5920,2021": [1723, 814],
+	"5919,2022": [1723, 814],
+	"5920,2022": [1723, 814],
+	// Yew Orc Fort Exit
+	"5654,1791": [729, 1451],
+	"5655,1791": [729, 1451],
+	"5654,1792": [729, 1451],
+	"5655,1792": [729, 1451]
+};
+
+function onUseChecked( pUser, iUsed )
+{
+	var socket = pUser.socket;
+	if( pUser.visible == 1 || pUser.visible == 2 )
+		pUser.visible = 0;
+
+	if( socket && iUsed && iUsed.isItem )
+	{
+		if( !iUsed.InRange( pUser, 3 ))
+		{
+			pUser.SysMessage( "I can't reach that." );
+			return false;
+		}
+		
+		// Use the tile coords as the lookup key
+		var key = iUsed.x + "," + iUsed.y;
+
+		// Try entrance map first
+		if( teleportFromMap( pUser, key, entranceMap ))
+			return true;
```

Comment:
"return true" here causes a system message that says "You can't think of a way to use that item." because it tries to pass the double-click event to code, and finds nothing actionable. Change to return false (as well as for the next one below.)